docs/v1/P4Runtime-Spec.mdk

```diff
@@ -687,9 +697,13 @@ follows:
 
 * The streaming channel between the controller and the server defines the
   liveness of the controller session. The controller is considered "offline" or
-  "dead" as soon as its corresponding stream channel to the switch is
-  broken, in which case the P4Runtime server quickly sets one of the slave
-  controllers with the highest `election_id` as master.
+  "dead" as soon as its corresponding stream channel to the switch is broken, in
+  which case the P4Runtime server will be without a master, until a client sends
```

Comment:
nit: "in which case the P4Runtime server will be without a master"...
This will only be true if the master channel is broken -- the preceding sentence fragment doesn't clarify that.
How about:
The controller is considered "offline" or "dead" as soon as its stream channel to the switch is broken. When a master channel gets broken: (i) an advisory message is sent to all other controllers for that `device_id` and `role_id`, as described in the [following section](#sec-mastership-notification) (ii) the P4Runtime server will be without a master, until a client sends a successful `MasterArbitrationUpdate` (as per the rules in a  [later section](#sec-mastership-updates)).

Reply:
Done.